x-pack/plugins/stack_alerts/server/alert_types/geo_containment/alert_type.ts

```diff
@@ -143,8 +143,9 @@ export interface GeoContainmentInstanceContext extends AlertInstanceContext {
   entityDocumentId: string;
   detectionDateTime: string;
   entityLocation: string;
-  containingBoundaryId: string;
-  containingBoundaryName: unknown;
+  // recovered alerts are not contained in boundary so context does not include boundary state
+  containingBoundaryId?: string;
```

Comment:
nit: May want to update the action variable description to say something "This value will not be set for recovered alerts"